### PR TITLE
fix: same presentational decimals everywhere in rfox stake

### LINKED ISSUE #7236: Differing decimal lengths in rFOX staking
### Overview

The amount of decimals used to show the amout of fox staked can change between screens in the staking process

### References and additional details

image

vs

image

### Acceptance Criteria

Should be same amount of decimals throughout the process 

### Need By Date

_No response_

### Screenshots/Mockups

_No response_

### Ownership

- [X] If my bounty needs engineering or needs product I have added the respective labels on the right
- [X] As the sponsor of this bounty I will review the changes in a preview environment (ops/product) or review the PR (engineering)

### Estimated effort

_No response_

### Sponsor / Stakeholder

_No response_

### Bounty Hunters

- [Join our discord](https://discord.gg/shapeshift)
- Include an expected timeline for you to complete work in the work plan when you apply for this bounty!
- Please refer to [this link](https://shapeshiftdao.gitbook.io/getting-started/how-to-contribute/bounties#resources-for-new-contributors-and-bounty-hunters) for some basic info
- Please ***do not*** start work on this issue until you are approved in Gitcoin.

### PR DESCRIPTION
**Pull request by NeOMakinG**
## Description
When you stake, the amount in the input should be the same as presented in the other components

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #7236

## Risk
Low

## Testing
- Use a number with 18 of precisions in the stake input
- Confirm and check if the numbers are rightly displayed

### Engineering
n/a

### Operations
n/a

## Screenshots (if applicable)
image
image
image

### CODE REVIEW

File: src/pages/RFOX/components/Stake/Bridge/BridgeConfirm.tsx [hunk 1]
```diff
@@ -127,15 +127,23 @@ export const BridgeConfirm: FC<BridgeRouteProps & BridgeConfirmProps> = ({ confi
           <Stack alignItems='center'>
             <AssetIcon size='sm' assetId={sellAsset?.assetId} />
             <Stack textAlign='center' spacing={0}>
-              <Amount.Crypto value={bridgeAmountCryptoPrecision} symbol={sellAsset.symbol} />
+              <Amount.Crypto
+                prefix='≈'
```
Comment (gomesalexandre) [inline src/pages/RFOX/components/Stake/Bridge/BridgeConfirm.tsx:131]: 💜

File: src/pages/RFOX/components/Stake/Bridge/BridgeConfirm.tsx [hunk 2]
```diff
@@ -204,7 +214,9 @@ export const BridgeConfirm: FC<BridgeRouteProps & BridgeConfirmProps> = ({ confi
             <TimelineItem>
               <CustomRow>
                 <Row.Label>{buyAsset?.name ?? ''}</Row.Label>
-                <Row.Value>{bridgeAmountCryptoPrecision}</Row.Value>
+                <Row.Value>
+                  <Amount.Crypto symbol='' prefix='≈' value={bridgeAmountCryptoPrecision} />
```
Comment (gomesalexandre) [inline src/pages/RFOX/components/Stake/Bridge/BridgeConfirm.tsx:218]: Same here 🙏🏽

**Review by gomesalexandre — COMMENTED** [review]
Tested locally, LGTM overall!

Two smolish omissions (see two last screenshots here) re: staking status precision, haven't been able to test bridge to status stage 

image
image
image
image
image
image
image
image
image

File: src/pages/RFOX/components/Stake/Bridge/BridgeConfirm.tsx (hunk 2 above)
Comment (NeOMakinG) [reply src/pages/RFOX/components/Stake/Bridge/BridgeConfirm.tsx]: Fixed in [91d7519](https://github.com/shapeshift/web/pull/7241/commits/91d75198190530c4b54f25776c78f4efd02293ba)

File: src/pages/RFOX/components/Stake/Bridge/BridgeConfirm.tsx [hunk 3]
```diff
@@ -174,7 +182,9 @@ export const BridgeConfirm: FC<BridgeRouteProps & BridgeConfirmProps> = ({ confi
             <TimelineItem>
               <CustomRow>
                 <Row.Label>{sellAsset?.name ?? ''}</Row.Label>
-                <Row.Value>{bridgeAmountCryptoPrecision}</Row.Value>
+                <Row.Value>
+                  <Amount.Crypto symbol='' prefix='≈' value={bridgeAmountCryptoPrecision} />
```
[the remark this answers is not in the document]
Comment (NeOMakinG) [reply src/pages/RFOX/components/Stake/Bridge/BridgeConfirm.tsx]: Fixed in [91d7519](https://github.com/shapeshift/web/pull/7241/commits/91d75198190530c4b54f25776c78f4efd02293ba)

File: src/pages/RFOX/components/Stake/Bridge/BridgeConfirm.tsx (hunk 1 above)
Comment (NeOMakinG) [reply src/pages/RFOX/components/Stake/Bridge/BridgeConfirm.tsx]: Fixed in [91d7519](https://github.com/shapeshift/web/pull/7241/commits/91d75198190530c4b54f25776c78f4efd02293ba)

**Review by gomesalexandre — APPROVED** [review]
gm https://jam.dev/c/166921ee-f37b-44d2-bb1f-ccf0c6223884

**Pull request merged**